rest/route/stats.go

```diff
@@ -531,20 +301,7 @@ func (tsh *taskStatsHandler) Parse(ctx context.Context, r *http.Request) error {
 }
 
 func (tsh *taskStatsHandler) Run(ctx context.Context) gimlet.Responder {
-	flags, err := evergreen.GetServiceFlags()
```

Comment:
I didn't think so. @ablack12 do you think we still need the service flag to disable task stats endpoint? Has it ever caused us an issue?

Reply:
I feel like we do just turn it off when things are bad because stats are expensive but idk if task_stats has been specifically the issue since we just turn off both. I think we might as well leave it if we can.